scripts/build/index.js

```diff
@@ -160,7 +160,7 @@ const saveServiceTags = () => {
 const updateServiceWorker = () => {
   const updateSWUrl = getApiUrl('/api/v2/upgrade/service-worker');
 
-  return rpn.get(updateSWUrl).catch(err => {
+  return request.get(updateSWUrl).catch(err => {
```

Comment:
This is just a name change with no actual change so it complicates the PR and git log. Please revert this and all the other changes in the scripts dir.

Reply:
Will do.